modules/installation-and-upgrade/pages/hardware-requirements.adoc

```diff
@@ -56,7 +56,11 @@ You can estimate how much space the [path]``/var/spacewalk/`` directory requires
 
 |
 | [path]``/var/spacewalk``
-| Minimum 50{nbsp}GB per {suse} product and 360{nbsp}GB per Red Hat product
```

Comment:
Minimum storage requirements: 50 GB for each SUSE product and 360 GB for each Red Hat product.